src/wmtk/TetMeshOperationExecutor.cpp

```diff
@@ -127,9 +129,32 @@ TetMesh::TetMeshOperationExecutor::TetMeshOperationExecutor(
         hash_update_region.add(v_closed_star);
     }
     hash_update_region.sort_and_clean();
+
+    global_simplex_ids_with_potentially_modified_hashes.resize(4);
     for (const simplex::Simplex& t :
          hash_update_region.simplex_vector(PrimitiveType::Tetrahedron)) {
         cell_ids_to_update_hash.push_back(m_mesh.id(t));
+
+        auto faces = wmtk::simplex::faces(m, t, false);
+        faces.add(t);
+        faces.sort_and_clean();
+
+        auto load = [&](PrimitiveType pt, size_t index) {
+            auto simps = faces.simplex_vector(pt);
```

Comment:
there's an idiosycnracy where simplex_vector(pt) is much slower than simplex_vector().
probably better to just go through simplex_vector() and insert into the right global_simplex..hashes for each one